### ISSUE TITLE
Not all DL article web sites have the software linked as supplementary material

### ISSUE THREAD

**Issue by bangerth** [ACM-TOMS/CALGO#11]
For example, article 1016 has no supplementary material: https://dl.acm.org/doi/10.1145/3446979 On the other hand, the paper for algorithm 905 does: https://dl.acm.org/doi/10.1145/1824801.1824812

Of course, all of the zip files that *should* be stored as supplementary material are available from here: https://calgo.acm.org/

We should make a sweep over all of the articles to see which ones have supplementary material linked to, and for those that don't, fix this. Separately, one could wonder whether the ones that do have supplementary material actually link to identical content to what's on https://calgo.acm.org/ .

**Comment by pauldj** [comment]
The workflow used to update the DL is unknown to Tim and Paolo. 
It looks like the DL is lagging behind with the supplementary material. 
- Tim will investigate
- Paolo can run a scan to identify which of the articles is missing the supplementary material

**Comment by pauldj** [comment]
Last night I ran a script to find all the articles that are missing the supplemental material. I woke up to discover that:
"Your IP Address has been blocked

Please contact dl-support@acm.org"

Obviously they did not like me to go through 1000 articles in rapid succession...

**Comment by bangerth** [comment]
Ha, that is funny :-) We talked to some ACM librarians a while ago. Did we ever hear back from them? Should we contact them again?

**Comment by pauldj** [comment]
Yes, I definitely think so. It would be good to understand their workflow. 

Btw, I just sent an email to Michelle G., hoping to get my IP whitelisted.

**Comment by pauldj** [comment]
Scripts are not allowed and the ACM staff, or maybe I should say the DL staff, absolutely refuses to collaborate. 
@bangerth : maybe you as EiC have more of a pull on these people. Otherwise it's something to bring up in the next meeting with ACM.
